cms/static/js/models/xblock_info.js

```diff
@@ -173,6 +173,10 @@ define(
              * True if summary configuration is enabled.
              */
                 summary_configuration_enabled: null,
+                /**
+             * List of tags of the unit. This list is managed by the content_tagging module.
+             */
```

Comment:
Don't copy the broken indentation above. Not sure why it's like that, but the asterisks in [JSDoc](https://jsdoc.app/) comments should always be aligned:

```
                /**
                 * List of tags of the unit. This list is managed by the content_tagging module.
                 */
                tags: null,
```